application/src/main/java/org/togetherjava/tjbot/commands/help/HelpSystemHelper.java

```diff
@@ -274,6 +274,10 @@ private void executeUncategorizedAdviceCheck(long threadChannelId, long authorId
                 return new CompletedRestAction<>(jda, null);
             }
 
+            if (threadChannel.isArchived()) {
+                return null;
```

Comment:
`return null` is the wrong action. this leads to the action crashing with an exception in JDA. instead, do it exactly like the `if` right before this one, with an already-completed action.

Reply:
yes, i wanted to ask this. Why i need to return something at all?? when the method returns void shouldn't it be just `return;`??

Reply:
ur not returning in the method. ur returning in the lambda thats used for the `flatMap` thats expecting a `RestAction<Message>`.

Reply:
ohh yea i see, thanks